### PR TITLE
[Tests-only] Fix ocis acceptance tests for new webdav api

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Fix the existing acceptance tests for new changes in ocis webdav API.

## Related Issue

- Related https://github.com/owncloud/ocis-reva/pull/77

## How Has This Been Tested?

:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiAuth/webDavAuth.feature [hunk 1]
```diff
@@ -8,17 +8,11 @@ Feature: auth
     When a user requests "/remote.php/webdav" with "PROPFIND" and no authentication
     Then the HTTP status code should be "401"
 
-  @skipOnOcis @issue-ocis-reva-62
+  @issue-ocis-reva-62
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuth/webDavAuth.feature]: Did the issue get fixed?
If so, do we remove the issue tag?

**Review by phil-davis — COMMENTED** [review]
LGTM - @individual-it please also review

File: tests/acceptance/features/apiAuth/webDavAuth.feature (hunk 1 above)
Comment (dpakach) [reply tests/acceptance/features/apiAuth/webDavAuth.feature]: It is fixed in one PR but not merged yet. What should we do about that?

Comment (phil-davis) [reply tests/acceptance/features/apiAuth/webDavAuth.feature]: We can merge this PR. It is easy enough to cleanup issue tags "some other time" (TM)

**Review by phil-davis — APPROVED** [review]

**Pull request merged**